ui/src/main.ts

```diff
@@ -17,3 +18,36 @@ setupCustomValidators();
 ReactModal.defaultStyles.overlay = {};
 ReactModal.defaultStyles.content = {};
 platformBrowserDynamic().bootstrapModule(AppModule);
+checkBrowserSupport();
+
+function checkBrowserSupport() {
```

Comment:
I think app component would probably be slightly more appropriate for this kind of initialization, unless there's a reason that won't work:

https://github.com/all-of-us/workbench/blob/2898b4b7cd2336fd529736dadf27301f581c31c7/ui/src/app/views/app/component.ts

Reply:
👍 to this. Maybe following the same pattern we used for `cookiesEnabled`